CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,6 @@
+Code of Conduct
+
+The nodejs/email repository is committed to upholding the Node.js Code of Conduct.
```

Comment:
Not an objection, but it's a bit odd describe the repository as being committed to upholding the CoC. I'd consider eliminating this sentence or rewording it.